WalletWasabi.Fluent/ViewModels/AddWallet/WelcomePageViewModel.cs

```diff
@@ -46,7 +49,27 @@ public WelcomePageViewModel(AddWalletPageViewModel addWalletPage)
 
 	public Bitmap AnonymousImage { get; } = AssetHelpers.GetBitmapAsset($"avares://WalletWasabi.Fluent/Assets/WelcomeScreen/{ThemeHelper.CurrentTheme}/anonymous.png");
 
-	public ICommand GetStartedCommand { get; }
+	private void OnNext()
+	{
+		if (SelectedIndex < NumberOfPages - 1)
+		{
+			SelectedIndex++;
+		}
+		else if (!Services.WalletManager.HasWallet())
+		{
+			Navigate().To(_addWalletPage);
+		}
+		else
+		{
+			Close();
+		}
+	}
 
-	public ICommand PrevCommand { get; }
+	private void OnBack()
+	{
+		if (SelectedIndex > 0)
+		{
+			SelectedIndex--;
+		}
+	}
```

Comment:
Leftover?

Reply:
@soosr I just removed the leftover code. I kept the right key navigation because I think it's cool ;)